README.md

```diff
@@ -1 +1,33 @@
-# Hello_Microverse
\ No newline at end of file
+# Hello-Microverse
```

Comment:
- [ ] Kindly use the [readme template](https://github.com/microverseinc/readme-template) provided by microverse and edit all sections. You should use all sections and only delete sections that do not apply to your project.